lam4doc/entrypoints/api/openapi/lam.yml

```diff
@@ -47,4 +59,21 @@ paths:
         200:
           description: OK.
         5XX:
-          description: Unexpected Error.
\ No newline at end of file
+          description: Unexpected Error.
+
+  /upload-rdf:
+    post:
+      operationId: lam4doc.entrypoints.api.handlers.upload_rdfs
+      description: Endpoint to upload RDF files.
+      requestBody:
+        required: false
+        content:
+          multipart/form-data:
+            schema:
+              $ref: '#/components/schemas/UPLOAD_SCHEMA'
```

Comment:
Mihai was mentioning that there is some sort of issue/bug with using schemas (which, I would strongly encourage) but it seems to be badly implemented in connextion. Check with him what is the best way of doing this.

Reply:
I did this with Mihai.